src/main/java/com/internet/shop/dao/jdbc/DriverDaoJdbcImpl.java

```diff
@@ -0,0 +1,114 @@
+import com.internet.shop.exception.DataProcessingException;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Driver;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class DriverDaoJdbcImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createQuery = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(createQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, driver.getName());
+            statement.setString(2, driver.getLicenceNumber());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create " + driver + " in DB", e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getQuery = "SELECT * FROM drivers WHERE id =? AND deleted = FALSE";
+        Driver driver = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getQuery)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                driver = createDriver(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get driver id + " + id + " from DB", e);
+        }
+        return Optional.ofNullable(driver);
+    }
+
+    @Override
+    public Driver update(Driver driver) {
+        String updateQuery = "UPDATE drivers SET name = ?, license_number = ? "
+                + "WHERE id = ? AND deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(updateQuery)) {
+            statement.setString(1, driver.getName());
+            statement.setString(2, driver.getLicenceNumber());
+            statement.setLong(3, driver.getId());
+            statement.executeUpdate();
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update " + driver + " in DB", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteQuery = "UPDATE drivers SET deleted = ?  WHERE id = ?";
```

Comment:
```suggestion
        String deleteQuery = "UPDATE drivers SET deleted = TRUE WHERE id = ?";
```
please don't set this boolean into this statement - there is no need for this